### PR TITLE
Fix #1293 : Telegram messages should be formatted with the right user unit preferences

### LINKED ISSUE #1293: Gladys answers in °F for Telegram temperature request
https://community.gladysassistant.com/t/configuration-unite-de-temperature-via-telegram/6512/5

### PR DESCRIPTION
**Pull request by Pierre-Gilles**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [ ] If your changes affects code, did your write the tests?
- [ ] Are tests passing? (`npm test` on both front/server)
- [ ] Is the linter passing? (`npm run eslint` on both front/server)
- [ ] Did you run prettier? (`npm run prettier` on both front/server)
- [ ] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Please provide a description of the change here. It's always best with screenshots, so don't hesitate to add some!

### CODE REVIEW

File: server/lib/user/user.getByTelegramUserId.js [hunk 1]
```diff
@@ -14,13 +14,25 @@ const { NotFoundError } = require('../../utils/coreErrors');
 async function getByTelegramUserId(telegramUserId) {
   const user = await db.User.findOne({
     where: { telegram_user_id: telegramUserId },
-    attributes: ['id', 'language'],
-    raw: true,
```
Comment (atrovato) [inline server/lib/user/user.getByTelegramUserId.js:18]: No raw anymore?

**Review by atrovato — APPROVED** [review]
Looks good.
A simple question, why getting some attributes and not entire user?

File: server/lib/user/user.getByTelegramUserId.js (hunk 1 above)
Comment (Pierre-Gilles) [reply server/lib/user/user.getByTelegramUserId.js:18]: I just .get({ plain: true }); after to get the created_at as native JS object like in the getUserById

**Comment by Pierre-Gilles** [comment]
> A simple question, why getting some attributes and not entire user?

@atrovato Because the entire user contains the password hash, we don't want it there :)

**Pull request merged**